### ISSUE TITLE
Why combined INSTRUCTION + INPUT + TARGET as INPUT for training ?!?!?

### ISSUE THREAD

**Issue by thusinh1969** [ymcui/Chinese-LLaMA-Alpaca#793]
### Check before submitting issues

- [X] Make sure to pull the latest code, as some issues and bugs have been fixed.
- [X] Due to frequent dependency updates, please ensure you have followed the steps in our [Wiki](https://github.com/ymcui/Chinese-LLaMA-Alpaca/wiki)
- [X] I have read the [FAQ section](https://github.com/ymcui/Chinese-LLaMA-Alpaca/wiki/FAQ) AND searched for similar issues and did not find a similar problem or solution
- [X] Third-party plugin issues - e.g., [llama.cpp](https://github.com/ggerganov/llama.cpp), [text-generation-webui](https://github.com/oobabooga/text-generation-webui), [LlamaChat](https://github.com/alexrozanski/LlamaChat), we recommend checking the corresponding project for solutions
- [X] Model validity check - Be sure to check the model's [SHA256.md](https://github.com/ymcui/Chinese-LLaMA-Alpaca/blob/main/SHA256.md). If the model is incorrect, we cannot guarantee its performance

### Type of Issue

Model training and fine-tuning

### Base Model

LLaMA-7B

### Operating System

Linux

### Describe your issue in detail

for s,t in zip(tokenized_sources['input_ids'],tokenized_targets['input_ids']):       
     input_ids = torch.LongTensor(s + t)[:max_seq_length] 

Why the original repo combined "Instruct + Input" + "target" as overall INPUT ? Target should be at labels, or is there something I missed ?

Steve

### Dependencies (must be provided for code-related issues)

```
# Please copy-and-paste your dependencies here.
```

### Execution logs or screenshots

```
# Please copy-and-paste your logs here.
```
